scripts/src/test/org/oppia/android/scripts/RegexPatternValidationCheckTest.kt

```diff
@@ -0,0 +1,136 @@
+package org.oppia.android.scripts
+
+import java.io.File
+import org.oppia.android.scripts.RegexPatternValidationCheck
+import org.junit.Test
+import org.junit.Rule
+import org.junit.Before
+import org.junit.After
+import org.junit.rules.TemporaryFolder
+import org.oppia.android.scripts.ScriptResultConstants
+import com.google.common.truth.Truth.assertThat
+import org.oppia.android.testing.assertThrows
+import java.io.PrintStream
+import java.io.ByteArrayOutputStream
+
+class RegexPatternValidationCheckTest {
+
+  @Rule
+  @JvmField
+  public var tempFolder: TemporaryFolder = TemporaryFolder()
+
+  @Before
+  fun initTestFilesDirectory() {
+    tempFolder.newFolder("testfiles")
+  }
+
+  private val outContent: ByteArrayOutputStream = ByteArrayOutputStream()
+  private val originalOut: PrintStream = java.lang.System.out
+
+  @Before
+  fun setUpStreams() {
+    java.lang.System.setOut(PrintStream(outContent))
+  }
+
+  @After
+  fun restoreStreams() {
+    java.lang.System.setOut(originalOut)
+  }
+
+  @Test
```

Comment:
Add a screenshot showing all these test cases are passing on local.

Reply:
okay, done.

Reply:
The screenshot says executed 0 out of 1 test passes. We have a total of 5 tests, could you confirm and check if all the 5 tests are passing on Bazel or not?

Reply:
I am not sure why this is showing this, but here `OK (5 tests)` is in the screenshot. I have also confirmed running the tests locally, all of them are passing on Bazel.